index.html

```diff
@@ -8,51 +8,58 @@
-                     a project you need coded,<br> 
-                    don’t hestiate to contact me.
-                </p>
-            </div>
-            <div class="social-media-class">
-                <ul class="social-media">
-                    <li class="social-media-links"><a href="#" class="LinkedIn-profile"><img src="./Images/Linkedin-icon.svg" alt="LinkedIn logo" class="linkedIn-class"></a></li>
-                    <li class="social-media-links"><a href="#" class="Twitter-profile"><img src="./Images/twitter.svg" alt="Twitter logo" class="Twitter-class"></a></li>
-                    <li class="social-media-links"><a href="#" class="Medium-profile"><img src="./Images/medium.svg" alt="Medium logo" class="Medium-class"></a></li>
-                    <li class="social-media-links"><a href="#" class="GitHub-profile"><img src="./Images/github.svg" alt="Github logo" class="Github-class"></a></li>
-                    <li class="social-media-links"><a href="#" class="Angellist-profile"><img src="./Images/angellist.svg" alt="Angellist logo" class="Angellist-class"></a></li>
-                </ul>
-            </div>
-        </div>
+        <section class="description-container">
+            <h1 class="description-title">
+                Hey there I'm<br>
+                Keenan<br>
+                <span>I'm a software<br>
+                developer</span>
+            </h1>
+            <p class="description-content">
+                I can help you build a product, <br>
+                    feature or website Look through<br>
+                    some of my work and experience!<br> 
+                    If you like what you see and have<br>
+                    a project you need coded,<br> 
+                don’t hestiate to contact me.
+            </p>
+            <ul class="social-media">
+                <li class="social-media-links"><a href="#" class="LinkedIn-profile"><img src="./Images/Linkedin-icon.svg" alt="LinkedIn logo" class="linkedIn-class"></a></li>
+                <li class="social-media-links"><a href="#" class="Twitter-profile"><img src="./Images/twitter.svg" alt="Twitter logo" class="Twitter-class"></a></li>
+                <li class="social-media-links"><a href="#" class="Medium-profile"><img src="./Images/medium.svg" alt="Medium logo" class="Medium-class"></a></li>
+                <li class="social-media-links"><a href="#" class="GitHub-profile"><img src="./Images/github.svg" alt="Github logo" class="Github-class"></a></li>
+                <li class="social-media-links"><a href="#" class="Angellist-profile"><img src="./Images/angellist.svg" alt="Angellist logo" class="Angellist-class"></a></li>
+            </ul>
+        </section>
         <!-- description container ends -->
       </div>
       <!-- Menu and about page start -->
-    <div class="menu-list">
+    <section class="menu-list">
         <div class="works-heading">
```

Comment:
Kindly note that the size of button increases too much when you hover on it, kindly fix that 🤗

![image](https://user-images.githubusercontent.com/73714615/187661472-84f3146e-da2b-4bee-ac42-7293a5e58515.png)